src/cfml/system/modules_app/server-commands/commands/server/prune.cfc

```diff
@@ -0,0 +1,159 @@
+        Boolean list        = false,   
+        Boolean force       = false,
+    ){
+        var serverInfo      = serverService.resolveServerDetails( arguments ).serverinfo;
+        var servers         = serverService.getServers();
+        var filterServers   = [];
+
+        print.line( "prune started...", 'yellow' ).toConsole();
+        if ( !arguments.force ){
+            servers.each( function( ID ){ runningServerCheck( servers[ arguments.ID ] ); } );
+        }
+
+        if( arguments.list )
+        {
+
+            generatePruneListServers( arguments.months, servers );
+        }
+        else
+        {
+
+            generatePruneListServers( arguments.months, servers );
+
+            var askContinuePrune = "Prune will forget and delete this servers! Do you still want to continue [y/n]?";
+
+            if( arguments.force || confirm( askContinuePrune ) ){
+
+                for( currentServer in servers ){
+                    result = dateDiff( "m", servers[ currentServer ].dateLastStarted, now() );
+                    if ( result >= arguments.months ){
+                        arrayAppend( filterServers, servers[ currentServer ] );
+                    }
+                }            
+
+                if ( arguments.force ){
+
+                    print.line( "using prune with force...", 'yellow' ).toConsole();
+
+                    var runningServers = getRunningServers( servers );
+                    /* areThereRunningServers = ( ! runningServers.isEmpty() ) */
+                    /* print.line( "are there running servers: #areThereRunningServers#" ).toConsole(); */
+
+                    if ( !runningServers.isEmpty() ) {
+
+                        var stopMessage = "Stopping server #serverInfo.name# first....";
+        
+                        print.line( stopMessage )
+                            .toConsole();
+        
+                        runningServers.each( function( ID ){
+                            var stopResults = serverService.stop( runningServers[ arguments.ID ] );
+                            print.line( stopResults.messages, ( stopResults.error ? 'red' : 'green' ) )
+                                .toConsole();
+                        } );
+        
+                        // Give them a second or three to die or file locks will still be in place (on Windows, at least)
+                        // This is hacky, but there's no clean way to poll for when the process is 100% dead
+                        sleep( 3000 );
+                    }
+
+                    for( currentServer in filterServers ){
+
+                        print.line( serverService.forget( #currentServer.name# ), 'red' )
+                            .toConsole();
+                    }
+
+                } else {
+                    
+                    for( currentServer in filterServers ){
+
+                        var askMessage = "Really forget & delete server '#currentServer.name#' forever [y/n]?";
+            
```

Comment:
This logic is already present in the `server forget` command.  Why is it duplicated here?